lib/tower_cli/resources/job.py

```diff
@@ -209,6 +209,14 @@ def launch(self, job_template=None, monitor=False, timeout=None,
         if supports_job_template_launch and not use_job_endpoint:
             job_id = job_started.json()['job']
 
+        # If 'ignored_fields' is present in the returning json, display it
+        # in verbose mode.
+        if 'ignored_fields' in job_started.json():
```

Comment:
The wording is fine, but I'm still not a fan of this implementation. Launching jobs are presumably the most frequently used part of tower-cli, and this is noise in the console output that shouldn't be necessary. If it's a normal job and everything is working fine, then I would opt to not show the message. It is more complicated to do it, but the logic can be added in the one-line conditional that you already have to deal with both 2.4.5 and 3.0 cases. i.e. `if 'ignored_fields' in job_started.json() and 'extra_vars' in job_started.json()['ignored_fields'] and job_started.json()['ignored_fields']['extra_vars']:`. That's not best-practice (you might want to store the JSON-ified version before that line), but the basic logic isn't unreasonable.